compose-settings-ui-m3/src/main/kotlin/com/alorma/compose/settings/ui/internal/SettingsTileScaffold.kt

```diff
@@ -77,6 +81,9 @@ internal fun SettingsTileScaffold(
         }
       }
     },
+    colors = ListItemDefaults.colors(
```

Comment:
Hi!

I think you can use `contentColorOf` to define at least the default content color on there, so it matches contrast specifications.

It should be added as param on the composables, or create some structure as `ListItemColors`

Reply:
But doinng something like 
```kotlin
colors = ListItemDefaults.colors(
      containerColor = MaterialTheme.colorScheme.contentColorFor(containerColor),
    )
```
the color would not match with the desired color, but maybe I missunderstood, the issue.

Reply:
I mean:

```
@[Composable]
colors(
    containerColor: ,
    headlineColor: contentColorOf(containerColor),
    leadingIconColor: ,
    overlineColor:,
    supportingColor: ,
    trailingIconColor:,
    disabledHeadlineColor:,
    disabledLeadingIconColor: ,
    disabledTrailingIconColor: ,
)
```

For all the non `containerColor` colors